Templates/Alerts/Sql/sql-deadlocks-per-second/azuredeploy.json

```diff
@@ -66,7 +65,13 @@
                     },
                     "trigger": {
                         "thresholdOperator": "GreaterThan",
-                        "threshold": 0
+                        "threshold": "[parameters('alertThreshold')]",
+                        "metricTrigger": {
+                            "thresholdOperator": "GreaterThan",
+                            "threshold": 3,
```

Comment:
Does an alert fires when Number of deadlocks/second > 0 for 3 separate 5m windows? Or when the Number of deadlocks/sec > 3?